### PR TITLE
add socket mode example link to README

### LINKED ISSUE #1016: Add link to Socket Mode example in README.md
Hi, I propose linking to the Socket Mode example from the README.md file. As I understand, Socket Mode is preferable to RTM, and I got confused between the two. See https://github.com/slack-go/slack/pull/1015 for suggested modification.

See https://api.slack.com/rtm for note:  "For most applications, Socket Mode is a better way to communicate with Slack."

Thanks.

### PR DESCRIPTION
**Pull request by itayd**
As mentioned in the documentation in https://api.slack.com/rtm:
"For most applications, Socket Mode is a better way to communicate with Slack."

Personally, I got confused between RTM and Socket Mode, and I think this might help future developers.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,8 +70,14 @@ func main() {
 }
 ```
 
+## Minimal Socket Mode usage:
+
+See https://github.com/slack-go/slack/blob/master/examples/socketmode/socketmode.go
+
 ## Minimal RTM usage:
 
+(For most applications, Socket Mode is a better way to communicate with Slack)
```
Comment (kanata2) [inline README.md]: Please add a link to the official announcement.

Comment (itayd) [reply README.md]: Done. Note that I couldn't link to the exact note but to the page that mentioned it. It's very prominent there though.

**Review by kanata2 — APPROVED** [review]
Thanks!

**Comment by itayd** [comment]
Hey @kanata2 , thanks for approving! Is there anything that delays merging? Wondering if I'm missing any steps...

**Comment by kanata2** [comment]
Sorry for the delay. I'll merge this.

**Pull request merged**